orca-api/src/main/java/com/netflix/spinnaker/orca/api/pipeline/models/StageExecution.java

```diff
@@ -123,6 +123,11 @@ public interface StageExecution {
 
   void setLastModified(@Nullable LastModifiedDetails lastModified);
 
+  @Nullable
+  Map<String, String> getMetricTags();
+
+  void setMetricTags(@Nullable Map<String, String> metricTags);
```

Comment:
We don't need to allow a null argument here, do we?

Reply:
yes, removed nullable from  method parameter